compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```
@@ -301,3 +301,5 @@ error.missing.arg.within.parenthesis=missing arg within parenthesis
 error.missing.positional.arg.in.error.constructor=missing positional arg in error constructor
 error.open.record.cannot.contain.rest.field=open record cannot contain rest field
 error.complex.variable.must.be.initialized=complex variable must be initialized
+error.isolated.var.cannot.be.declared.as.public=isolated variable cannot be declared as public
```

Shall we say 
```suggestion
error.isolated.var.cannot.be.public=isolated variable cannot be public
```
similar to the next error?